### DISCUSSION TITLE
Should we rename this project to maplibre-gl-ts?

### DISCUSSION THREAD

**Discussion by hy9be** [maplibre/maplibre-gl-js#1386] [Polls]
Curious about how everybody thinks about this. `maplibre-gl-js` is actually a TypeScript project. Should we rename it?

**Comment by nyurik** [comment]
I would say no because of the name recognition. The less confusion there is, the better. Besides, eventually I hope it will all be a single `maplibre` project (based on `maplibre-rs` cross-compiled into everything :) )

P.S.  Note that in NPM, it is called [maplibre-gl](https://www.npmjs.com/package/maplibre-gl). This is also a bit confusing, because technically the native is also gl. Alas. :shrug: 
![image](https://user-images.githubusercontent.com/1641515/180358148-8fd8c9c5-cd5a-49ec-bd55-d99ee4921761.png)

**Comment by wipfli** [comment]
I prefer MapLibre GL JS...

**Comment by HarelM** [comment]
The browser still runs javascript :-) but in general I don't see an actual need to change the library name...
